.gitignore

```diff
@@ -4,3 +4,4 @@
 Gemfile.lock
 pkg/*
 doc
+.ruby-version
```

Comment:
Why ignore this file?

Reply:
🤷 I figured one cuz it's a gem in here, so I didn't want to give the impression it requires a specific version of ruby. And two I'm going with a policy of minimal intrusion on these gems until we decide what to do with them.